src/components/pages/GamePlay/GameDrawStep.js

```diff
@@ -134,13 +135,13 @@ export default function GameDrawStep(props) {
                 onClick={handleSubmit}
                 loading={isUploading}
               >
-                {isUploading ? 'Uploading...' : 'Save'}
+                {isUploading ? 'Uploading...' : 'Submit'}
               </Button>
             )}
 
-            <button className="skip-btn" onClick={handleSkip}>
+            <Button className="skip-btn" onClick={handleSkip}>
```

Comment:
Good job implementing antd buttons to make them stand out more. It might be a good idea to explain this with comments in the code. I didn't realize this was what you were doing until I read the description and thought you had made some mistake by capitalizing button.